scripts/build.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+pwd
+cd scripts
+
+#add "f" to force create env
+bash setup_env.sh $1
+cd ../..
+
+ENV_NAME=env_sphinx
+pwd
+source $ENV_NAME/bin/activate
+
+#clone repos
+for repo_name in GenAIComps GenAIEval GenAIExamples GenAIInfra opea-project.github.io; do
+  echo "prepare for $repo_name"
+
+  if [[ "$1" == "f" ]]; then
+    echo "force to clone rep ${repo_name}"
+    rm -rf ${repo_name}
+  fi
+
+  if [ ! -d ${repo_name} ]; then
+    URL=https://github.com/opea-project/${repo_name}.git
+
+    echo "git clone $URL"
+    git clone $URL
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      echo "git clone ${repo_name} is wrong, try again!"
+      rm -rf ${repo_name}
+      exit 1
+    fi
+    sleep 10
+  else
+    echo "found existed repo folder ${repo_name}, skip to clone"
+  fi
+done
+
+echo "Build HTML"
+cd docs
+make clean
+make html
+echo "Build online doc done!"
+
+echo "update github.io"
+
+RELEASE_FOLDER=../opea-project.github.io
+BUILDDIR=_build
+PUBLISHDIR=${RELEASE_FOLDER}/latest
+
+cp -r ${BUILDDIR}/html/*  ${PUBLISHDIR}
```

Comment:
@NeoZhangJianyu @mkbhanda @yinghu5 @chensuyue You left out a step that's in the "make publish" steps that handles the case when a file was deleted from the previously published documents.  You should remove everything from the PUBLISHDIR folder before copying al the files that were just built:

```
rm -rf $(PUBLISHDIR)/*
```

Reply:
Yes, I will update as your comment later.

Thank you!